src/util/config/files.js

```diff
@@ -69,7 +70,19 @@ export function getAuthConfigFilePath() {
 }
 
 export function readLocalConfig(prefix) {
-  const target = getLocalPathConfig(prefix || process.cwd());
+  let target;
+
+  try {
+    target = getLocalPathConfig(prefix || process.cwd());
+  } catch (err) {
+    if (err instanceof NowError) {
+      console.error(error(err.message));
```

Comment:
Leveraging `err.message` here because it can be trusted and involves enough logic constructing the error message to DRY it up. This is in accordance with @rauchg's plan to reuse error messages instead of reconstructing them every time.

Reply:
👍 Let's use it